.github/workflows/python-bindings.yml

```diff
@@ -0,0 +1,36 @@
+# This file is autogenerated by maturin v1.2.2
+# To update, run
+#
+#    maturin generate-ci github
+#
+name: Python bindings CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+    paths:
+      - 'py-rattler/**/*'
```

Comment:
Should we also add this file itself to the list of changed paths?

Reply:
Done